deepchem/molnet/load_function/qm9_datasets.py

```diff
@@ -69,25 +72,28 @@ def load_qm9(featurizer='CoulombMatrix', split='random', reload=True):
         tasks=qm9_tasks, smiles_field="smiles", featurizer=featurizer)
 
   dataset = loader.featurize(dataset_file)
-  splitters = {
-      'index': deepchem.splits.IndexSplitter(),
-      'random': deepchem.splits.RandomSplitter(),
-      'stratified':
-      deepchem.splits.SingletaskStratifiedSplitter(task_number=11)
-  }
-  splitter = splitters[split]
-  train_dataset, valid_dataset, test_dataset = splitter.train_valid_test_split(
-      dataset)
-  transformers = [
-      deepchem.trans.NormalizationTransformer(
-          transform_y=True, dataset=train_dataset)
-  ]
-  for transformer in transformers:
-    train_dataset = transformer.transform(train_dataset)
-    valid_dataset = transformer.transform(valid_dataset)
-    test_dataset = transformer.transform(test_dataset)
+  if split == None:
+    raise ValueError()
+  else:

```

Comment:
This else falls into the same category as before.  There is no need for it as the raise from the if already exits the method.